CONTRIBUTING.md

```
@@ -90,7 +92,9 @@ cat /tmp/numbast-v<NEW_VERSION>-changelog.txt >> /tmp/numbast-v<NEW_VERSION>-tag
 git tag -a v<NEW_VERSION> -F /tmp/numbast-v<NEW_VERSION>-tag.txt
 git push origin v<NEW_VERSION>
 ```
-7. Verify docs deployment publishes `https://nvidia.github.io/numbast/<NEW_VERSION>/` and that `<NEW_VERSION>` appears in the docs version picker.
+7. Verify the publish workflow uploads PyPI wheels.
```

markdownlint-cli2 (0.23.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 7; Style: 1/2/3

(MD029, ol-prefix)